docker/image/whyis-server/celeryd

```diff
@@ -0,0 +1,22 @@
+# Name of nodes to start
```

Comment:
Should be in whyis-server-deps image directory, since that's where it's incorporated?